External/FEXCore/Source/Utils/FileLoading.h

```diff
@@ -14,5 +14,16 @@ namespace FEXCore::FileLoading {
    * @return true on file loaded, false on failure
    */
   bool LoadFile(std::vector<char> &Data, const std::string &Filepath, size_t FixedSize = 0);
+
+  /**
+   * @brief Loads a filepath in to a buffer of data with a fixed size
```

Comment:
I like the general idea, but not being able to handle files smaller than FixedSize makes this a bad abstraction IMO.

Reply:
It now returns size read instead of bool.